ale/drivers/messenger_drivers.py

```diff
@@ -150,27 +169,33 @@ def detector_center_sample(self):
         Returns center detector sample acquired from Spice Kernels.
         Expects ikid to be defined. This should be the integer Naid ID code for
         the instrument.
+        
+        NOTE: These values are defined in an ISIS iak file
 
         Returns
         -------
         : float
           center detector sample
         """
-        return float(spice.gdpool('INS{}_BORESIGHT'.format(self.ikid), 0, 3)[0])
+        # return float(spice.gdpool('INS{}_BORESIGHT'.format(self.ikid), 0, 3)[0])
```

Comment:
Should we keep this commented line?

Reply:
Probably not? I left it in there for now to make sure I was handling this right. I can clean this up once confirmed.